services/hubspot.ts

```diff
@@ -0,0 +1,27 @@
+interface ISendEvent {
+  data: {
+    [property: string]: string;
+  };
+  email: string;
+}
+
+const updateContact = async ({ data, email }: ISendEvent) => {
+  const options = {
+    method: "POST",
+    qs: { hapikey: process.env.HUBSPOT_API_KEY },
```

Comment:
@zach-at-the-skills - We started access tokens using HubSpot private apps for authentication instead of the generic HubSpot API Key. Can you create private apps for Vulcan Staging and Vulcan Production and update Vercel with the respective access tokens?

See here for our current private apps using by our other repos: https://app.hubspot.com/private-apps/6584378

Reply:
Looks like we'll need to refactor the Thetis implementation
https://github.com/skills-annex/thetis/blob/master/src/pages/api/hs-proxy.ts#L27